modules/reliability/php/NDB_Menu_Filter_reliability.class.inc

```diff
@@ -310,7 +310,7 @@ class NDB_Menu_Filter_reliability extends NDB_Menu_Filter
     function _populate_reliability_table()
     { 
        $db =& Database::singleton();
-       $this->reliability_table= $db->pselect("SELECT CommentID, reliability_center_id FROM reliability");
+       $this->reliability_table= $db->pselect("SELECT CommentID, reliability_center_id FROM reliability", array());
```

Comment:
Yes, this change is not related, but I as watch the logs I fixed this missing array.  I will mention it in the pull request description.